src/net/types.rs

```diff
@@ -1436,7 +1436,7 @@ bitflags! {
         const NONBLOCK = bitcast!(c::SOCK_NONBLOCK);
 
         /// `SOCK_CLOEXEC`
```

Comment:
It would be good to add a documentation comment here mentioning that on Windows, `CLOEXEC` is implemented by way of `WSA_FLAG_NO_HANDLE_INHERIT`.